OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove this change from the commit

Reply:
These changes happens all the time when the csproj files are updated with VS2017, so there is something VS is not happy about and reformatting the file. Maybe we should accept these changes so we dont have to think about them anymore?